src/sdk/lsf-sdk.js

```diff
@@ -539,8 +539,9 @@ export class LSFWrapper {
   // Proxy events that are unused by DM integration
   onEntityCreate = (...args) => this.datamanager.invoke("onEntityCreate", ...args);
   onEntityDelete = (...args) => this.datamanager.invoke("onEntityDelete", ...args);
-  onSelectAnnotation = (...args) =>
-    this.datamanager.invoke("onSelectAnnotation", ...args);
+  onSelectAnnotation = (...args) => {
+    this.datamanager.invoke("onSelectAnnotation", ...[...args, this]);
```

Comment:
very confusing, by the code it looks like there are can be any number of arguments, but actually we expect prev and next only and always, or it'll break. so there is should be explicit list of arguments

Reply:
or move LSF instance to be the first argument